addon/components/marker-layer.js

```diff
@@ -2,6 +2,7 @@ import BaseLayer from 'ember-leaflet/components/base-layer';
 import DraggabilityMixin from 'ember-leaflet/mixins/draggability';
 import PopupMixin from 'ember-leaflet/mixins/popup';
 import toLatLng from 'ember-leaflet/macros/to-lat-lng';
+import observer from 'ember-metal/observer';
```

Comment:
Let's use `const { observer } = Ember;` like we do on the rest of the project, please.
We'll update everything when the modules story is settled for good.